structuredefinitions/England-ServiceRequest.xml

```diff
@@ -205,10 +190,6 @@
     <element id="ServiceRequest.code.coding:SNOMEDProcedureCode">
       <path value="ServiceRequest.code.coding" />
       <sliceName value="SNOMEDProcedureCode" />
-      <binding>
-        <strength value="preferred" />
-        <valueSet value="https://fhir.hl7.org.uk/ValueSet/UKCore-ProcedureCode" />
```

Comment:
Based on your discussions with UK Core, this was was advised?

Reply:
Binding was removed to revert to present state in NHSDigital-ServiceRequest. 
Note: ServiceRequest.code.coding:SNOMEDProcedureCode has no binding. IOPS-2086 has been raised(needs DA discussion for resolution)